packages/evolution-generator/src/scripts/generate_survey.py

```diff
@@ -33,7 +34,28 @@ def generate_survey(config_path):
         # Get the data from the YAML file
         survey_folder_path = surveyGenerator["survey_folder_path"]
         excel_file_path = surveyGenerator["excel_file_path"]
-        enabled_scripts = surveyGenerator["enabled_scripts"]
+        enabled_scripts = surveyGenerator.get("enabled_scripts", [])
+
+        required_scripts = [
+            "generate_excel",
+            "generate_section_configs",
+            "generate_sections",
+            "generate_widgets_configs",
+            "generate_widgets",
+            "generate_conditionals",
+            "generate_choices",
+            "generate_input_range",
+            "generate_libelles",
+            "generate_UI_tests",
+        ]
+        missing_scripts = [
```

Comment:
Does it mean that you force the generate_UI_tests script? What will happen when you run it out of the box in MJ or LD?

Reply:
I will propose a more elegant solution on Tuesday.